src/page/Policy/components/UserInfoCard.tsx

```
@@ -6,54 +6,50 @@ import { useNavigate } from "react-router-dom";
 
-export default function UserInfo({
-  userName,
-  lineId,
-  isOwner = false,
-  planName,
-  isDualPhone = false,
-  sharedDataRemaining,
-  personalDataRemaining,
-  isBlocked = false,
-}: Props) {
+export default function UserInfo({ userData }: Props) {
```

Biome (2.4.6)

[error] Shouldn't redeclare 'UserInfo'. Consider to delete it or rename it.

(lint/suspicious/noRedeclare)